src/gui/widgets/Fader.cpp

```diff
@@ -433,4 +698,32 @@ void Fader::paintLevels(QPaintEvent* ev, QPainter& painter, bool linear)
 	painter.restore();
 }
 
+void Fader::paintFaderTicks(QPainter& painter)
+{
+	painter.save();
+
+	const QPen zeroPen(QColor(255, 255, 255, 216), 1.5);
+	const QPen nonZeroPen(QColor(255, 255, 255, 128), 1.);
+
+	for (float i = 6.f; i >= -120.f; i-= 6.f)
```

Comment:
This kinda feels like magic numbers to me. I feel like the interval should be specified in a single place (maybe in the `public` section of the `Fader` class) and then pulled from whenever it is needed.

Reply:
Commit f02c464c2f0 improves this by making sure that it paints correctly for all step sizes (except 0) and that it takes the maximum value of the model into account. It now also references the constant that describes the minimum dB value. So the magic numbers are gone now.